README.md

```diff
@@ -1,66 +1,55 @@
-# Pure
+# pure [![Fish Shell Version](https://img.shields.io/badge/fish-v2.2.0-007EC7.svg?style=flat-square)](http://fishshell.com)
```

Comment:
@edouard-lopez I really want to update this, so I'm going to move forward just because it's a simple change. Let me know if there are any concerns :+1: